### DISCUSSION TITLE
Problem flashing Sonoff Th Elite

### DISCUSSION THREAD

**Discussion by Hurlie** [arendst/Tasmota#22093] [General]
Hello,

I flashed a Sonoff Th Elite with NodeMuc Flasher with the actual Version "tasmota32.bin" 14.2.0 Rita on ESP32. 
(Output attached)

Everything seems to work fine.

After this, I have to power on the device, so it starts the AP-Mode. But after powering on the device, only the "auto" led goes on.

Normaly the Wifi LED has to flash and the AP can be viewed in the Wifi List.

But this does not happen:(

What goes wrong?

Do you have any suggestions?

![Flash](https://github.com/user-attachments/assets/0cc74388-fd5d-4f6f-89c7-030bf4b347fd)
![Version](https://github.com/user-attachments/assets/168e2a29-2a23-4bca-bf37-cbfcce80b03f)

Many thanks for your help.

**Comment by sfromis** [comment]
You didn't read the page you posted a screenshot from?

The file you highlighted is in the section "OTA Firmware" (meaning only for OTA), instead of "Factory binaries to be used for initial flashing", which is what you should have used.

And instead of messing with a non-supported tool for flashing (those "NodeMCU" tools can only be expected to work for ESP8266, not ESP32), it would have been simpler if you followed the link after "for an initial installation of Tasmota", which is a supported (and recommended) tool, automatically picking a binary (default selection) to match your hardware, obviously in accordance with the previous paragraph.

**Reply by Hurlie** [reply]
Hello sfromis,

Tnx for the answer. Your are right. I havent read the page. My mistake:(

Now I tried the link "for an initial installation of Tasmota".
That works fine. (See attached screenshots.)

Unfortunatly the result is the same. After attaching the TH Elite to power, there is no reaction from the device.
Only the "Auto" LED glimms.

I have tried this with two different TH Elite. Both with the same result:(

Thats bad. Do you have an idea, where I made the mistake?

Thnx in advance,

Hurlie



![3](https://github.com/user-attachments/assets/c87797dc-cf13-46b2-95be-b57044012b8c)
![1](https://github.com/user-attachments/assets/c89f02c6-b579-4839-8997-51b92a30889d)
![2](https://github.com/user-attachments/assets/7307dab2-314f-47ae-9a4c-c61714329135)

**Reply by sfromis** [reply]
Not clear what "reaction" you were expecting or looking for? Nothing will be visible on the device until the Tasmota configuration has been done; instead the expected step is that it will broadcast an AP on Wifi to connect to, for onboarding.

**Reply by Hurlie** [reply]
Like you say, I would expect, that the device would broadcast an AP with the Blue Wifi LED flashing. Then I should see this broadcasted WIFI Net on my phone. But this does not happen. 
The blue WIFI LED is not flashing and there is no Wifi visible on my phone.

I have tried this before I flashed the Tasmota. At this condition the LED flashes and the AP is visible,

**Reply by sfromis** [reply]
Well, don't expect a flashing led before configuring where it could flash. What I'd do would be to disconnect from mains, connect the USB/serial adapter, and then boot with a terminal program running to see the messages on serial.

**Reply by Hurlie** [reply]
Thx for your help:) I will try booting with a terminal progamm.

**Reply by Hurlie** [reply]
Hello sfromis,

I connected with HTERM over USB/Serial and got the following log.
Seems, as if there is a boot loop.
Do you have any suggestions, what goes wrong?


Project tasmota - Tasmota Version 14.2.0(release-tasmota32)-3_0_4(2024-08-14T12:39:45)<\r><\n>00:00:00.000
WIF: WifiManager active for 3 minutes<\r><\n>ets Jul 29 2019 12:21:46<\r><\n><\r><\n>rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)<\r><\n>
configsip: 0, SPIWP:0xee<\r><\n>clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00<\r><\n>mode:DIO, clock div:2<\r><\n>load:0x3fff0030,len:184<\r><\n>load:0x40078000,len:13228<\r><\n>ho 0 tail 12 room 4<\r><\n>
load:0x40080400,len:4<\r><\n>load:0x40080404,len:2924<\r><\n>entry 0x40080570<\r><\n><\r><\n>00:00:00.001 HDW: ESP32-D0WD-V3 v3.0 <\r><\n>00:00:00.015 UFS: FlashFS mounted with 308 kB free<\r><\n>00:00:00.026 CFG:
Loaded from File, Count 2<\r><\n>00:00:00.037 QPC: Count 5<\r><\n>00:00:00.585 BRY: Berry initialized, RAM used 5060 bytes<\r><\n>00:00:00.596 

Project tasmota - Tasmota Version 14.2.0(release-tasmota32)-3_0_4(2024-08-14T12:39:45)<\r><\n>00:00:00.000 WIF:

..
..
..

**Reply by Hurlie** [reply]
Hello stromis,

I connected over USB/serial with Hterm and got the following log:

Seems to be an boot loop.

Project tasmota - Tasmota Version 14.2.0(release-tasmota32)-3_0_4(2024-08-14T12:39:45)<\r><\n>00:00:00.000
WIF: WifiManager active for 3 minutes<\r><\n>ets Jul 29 2019 12:21:46<\r><\n><\r><\n>rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)<\r><\n>
configsip: 0, SPIWP:0xee<\r><\n>clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00<\r><\n>mode:DIO, clock div:2<\r><\n>load:0x3fff0030,len:184<\r><\n>load:0x40078000,len:13228<\r><\n>ho 0 tail 12 room 4<\r><\n>
load:0x40080400,len:4<\r><\n>load:0x40080404,len:2924<\r><\n>entry 0x40080570<\r><\n><\r><\n>00:00:00.001 HDW: ESP32-D0WD-V3 v3.0 <\r><\n>00:00:00.015 UFS: FlashFS mounted with 308 kB free<\r><\n>00:00:00.026 CFG:
Loaded from File, Count 2<\r><\n>00:00:00.037 QPC: Count 5<\r><\n>00:00:00.585 BRY: Berry initialized, RAM used 5060 bytes<\r><\n>00:00:00.596 

Project tasmota - Tasmota Version 14.2.0(release-tasmota32)-3_0_4(2024-08-14T12:39:45)<\r><\n>00:00:00.000 WIF:

**Reply by sfromis** [reply]
Well, a typical cause of a boot loop that way is if your USB/serial adapter is too weak (lots of those are, except for the minority able to act as a small power supply) to allow the ESP32 to boot to include Wifi, and the log does look like this is happening. While that *should* be no issue when using mains power instead, there is not really more to see there, except to confirm that Tasmota was flashed and able to start booting.

**Reply by Hurlie** [reply]
ok, thats sound possible. When connected to USB there is too less power.
So I have to look for another USB/Serial Adapter with enough power?
Do you have a suggestion, which adapter would fit?
Attached pictures of my actual one.


![PXL_20240906_091131127](https://github.com/user-attachments/assets/9ed8c81d-c963-40a5-98df-3f6f279b7dec)
![PXL_20240906_091157833](https://github.com/user-attachments/assets/fb22a5a1-6c98-4723-9cd9-f8dd94c76984)
![PXL_20240906_091201716](https://github.com/user-attachments/assets/fc35871c-d03b-4921-bea6-9652bb981f2b)
![PXL_20240906_091215597](https://github.com/user-attachments/assets/bd73cac8-b26c-4bd5-a8a8-f69375759cd3)

**Reply by sfromis** [reply]
Oh no, **that** board again :cry: It is indeed not much recommended as it is not capable of delivering good power for ESP32, unless you can add a good 3.3V power source on the side (with common ground).

Both [recommended](https://tasmota.github.io/docs/Getting-Started/#serial-programmer) ones are highlighted with yellow, the first one being the cheapest, but may still be borderline for getting a Wifi connection, depending on the board with the ESP32.

If you have a NodeMCU around, that can also be used as a pretty good USB/serial adapter, with the described wiring. And cheaper than the VoltLink board.

**Reply by Hurlie** [reply]
Hello sfromis,

tnx for your help an patience.

Ive just ordered the Voltlink Adapter. When its arrived, I will try it and tell you about the log:)

Have a nice day:)
